### PR TITLE
Windows compatibility updates

### PR DESCRIPTION
**Pull request by erinclemmer**
Also changed webpack.config.js to work with older versions of node

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -7,7 +7,8 @@ module.exports = [
   makeConfig({ minify: true })
 ]
 
-function makeConfig({ minify } = {}) {
```
Comment (erinclemmer) [inline webpack.config.js:10]: Older versions of node (including 5.11.0) cannot run this line

File: README.md [hunk 2]
```diff
@@ -63,4 +63,12 @@ ko.applyBindings()
...
 ```
 
+#### NPM Global Requirements
+Install these packages globally to use them  
+E.G. `npm install -g {package}`
+* babel
+* eslint
+* karma
+* webpack
+
```
Comment (erinclemmer) [inline README.md]: Need to display the global prereqs for development somewhere

**Comment by caseyWebb** [comment]
npm run scripts eliminate the need for global deps, and you can pass additional args as such `npm run test -- --browsers IE`

File: package.json [hunk 3]
```diff
@@ -16,6 +16,8 @@
     "stage": "git add dist example/dist lib",
     "test": "karma start",
     "test:watch": "karma start karma.watch.js",
+    "test-IE": "karma start --browsers IE",
+    "test-IE:watch": "karma start --browsers IE karma.watch.js",
```
Comment (erinclemmer) [inline package.json]: Needed the ability to test in IE

File: package.json [hunk 4]
```diff
@@ -7,7 +7,7 @@
   "repository": "Profiscience/ko-component-router",
   "main": "dist/ko-component-router.js",
   "scripts": {
-    "build": "npm run build:lib & npm run build:dist; wait",
+    "build": "npm run build:lib & npm run build:dist",
```
Comment (erinclemmer) [inline package.json]: wait does not have a command that translates to Windows

**Comment by caseyWebb** [comment]
The preferred method for this is going to be using bash on Windows. POSIX or bust.

**Comment by erinclemmer** [comment]
I think if you run `npm run XXX` it goes back to the default windows CLI
capture

**Comment by caseyWebb** [comment]
Try using the WSL rather than git bash, or changing the run script to `bash -c "..."`

**Pull request merged**